### PR TITLE
[gNMI-1.11, RT-5.1/3] - add InterfaceCountersUpdateDelayed deviation for CISCO

### PR DESCRIPTION
**Pull request by kjahed**
CISCO interface counters telemetry does not reflect the latest value. This PR adds InterfaceCountersUpdateDelayed and modify gNMI-1.10, gNMI-1.11, RT-5.1, and RT-5.3 to wait until the counters are updated (for a max timeout) when activated.

In addition, for gNMI-1.10, modify `verifyChassisIsAncestor` to ignore ancestors with no type. This is similar to PR #2808 for gNMI-1.4.

### CODE REVIEW

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go [hunk 1]
```diff
@@ -219,8 +219,9 @@ func TestHardwarePort(t *testing.T) {
 	t.Logf("For interface %s, HardwarePort is %s", dp.Name(), val)
 
 	// Verify HardwarePort is a component of type PORT.
-	typeGot := gnmi.Get(t, dut, gnmi.OC().Component(val).Type().State())
-	if typeGot != portType {
+	typeGotV := gnmi.Lookup(t, dut, gnmi.OC().Component(val).Type().State())
+	typeGot, present := typeGotV.Val()
+	if present && typeGot != portType {
```
Comment (LimeHat) [inline feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:224]: This change was not included in the PR description / not part of the deviation, it appears

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go [hunk 2]
```diff
@@ -400,8 +401,9 @@ func verifyChassisIsAncestor(t *testing.T, dut *ondatra.DUTDevice, comp string)
 			t.Errorf("Chassis component NOT found as an ancestor of component %s", comp)
 			break
 		}
-		got := gnmi.Get(t, dut, gnmi.OC().Component(val).Type().State())
-		if got == chassisType {
+		gotV := gnmi.Lookup(t, dut, gnmi.OC().Component(val).Type().State())
+		got, present := gotV.Val()
+		if present && got == chassisType {
```
Comment (LimeHat) [inline feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:406]: This change was not included in the PR description / not part of the deviation, it appears

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go (hunk 1 above)
Comment (kjahed) [reply feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go]: Thanks. Reverted.

File: feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go (hunk 2 above)
Comment (kjahed) [reply feature/gnmi/otg_tests/telemetry_basic_check_test/telemetry_basic_check_test.go:405]: Thanks. Updated the PR description.

**Comment by AmrNJ** [comment]
@kjahed Please can you resolve the conflicts?

**Review by AmrNJ — APPROVED** [review]
LGTM, Please resolve the conflicts to merge

**Comment by kjahed** [comment]
> LGTM, Please resolve the conflicts to merge

@AmrNJ done.

**Comment by AmrNJ** [comment]
@kjahed It still has conflicts and readme check fails. Please can you fix it?

**Comment by arvbaska1** [comment]
Merge conflicts are fixed, the Readme check fails due to some issue in test-requirements-template.md
https://github.com/openconfig/featureprofiles/actions/runs/10944362833/job/30386029201?pr=2805

Local check passes for this test
go run ./tools/validate_readme_spec/validate_readme_spec.go --feature-dir feature/interface/aggregate/otg_tests/balancing_test/README.md
 %

**Review by ram-mac — APPROVED** [review]
LGTM

**Pull request merged**